src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -589,25 +621,28 @@ NodeHasActiveShardPlacements(char *nodeName, int32 nodePort)
 	SysScanDesc scanDescriptor = NULL;
 	ScanKeyData scanKey[scanKeyCount];
```

Comment:
That's not going to work on all compilers - you can't size arrays by a variable amount.